sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+        )
+        return AsyncItemPaged(
+            get_next, extract_data)
+
+    async def get_subscription(
+            self, topic: Union[str, TopicDescription], subscription_name: str, **kwargs
+    ) -> SubscriptionDescription:
+        """Get a topic subscription.
+
+        :param Union[str, TopicDescription] topic: The topic that owns the subscription.
+        :param str subscription_name: name of the subscription.
+        :rtype: ~azure.servicebus.management.SubscriptionDescription
+        """
+        try:
+            topic_name = topic.name  # type: ignore
+        except AttributeError:
+            topic_name = topic
+        entry_ele = await self._get_subscription_element(topic_name, subscription_name, **kwargs)
+        entry = SubscriptionDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError(
+                "Subscription('Topic: {}, Subscription: {}') does not exist".format(subscription_name, topic_name))
+        subscription = SubscriptionDescription._from_internal_entity(entry.content.subscription_description)
+        subscription.name = entry.title
+        return subscription
+
+    async def get_subscription_runtime_info(
+            self, topic: Union[str, TopicDescription], subscription_name: str, **kwargs
+    ) -> SubscriptionRuntimeInfo:
+        """Get a topic subscription runtime info.
+
+        :param Union[str, TopicDescription] topic: The topic that owns the subscription.
+        :param str subscription_name: name of the subscription.
+        :rtype: ~azure.servicebus.management.SubscriptionRuntimeInfo
+        """
+        try:
+            topic_name = topic.name  # type: ignore
+        except AttributeError:
+            topic_name = topic
+        entry_ele = await self._get_subscription_element(topic_name, subscription_name, **kwargs)
+        entry = SubscriptionDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError(
+                "Subscription('Topic: {}, Subscription: {}') does not exist".format(subscription_name, topic_name))
+        subscription = SubscriptionRuntimeInfo._from_internal_entity(entry.content.subscription_description)
+        subscription.name = entry.title
+        return subscription
+
+    async def create_subscription(
+            self, topic: Union[str, TopicDescription], subscription: Union[str, SubscriptionDescription], **kwargs
+    ) -> SubscriptionDescription:
+        """
+
+        :param Union[str, TopicDescription] topic: The topic that will own the to-be-created subscription.
+        :param Union[str, ~azure.servicebus.management.SubscriptionDescription] subscription: The subscription name or a
+        `SubscriptionDescription` instance. When it's a str, it will be the name of the created subscription.
+         Other properties of the created subscription will have default values decided by the ServiceBus.
```

Comment:
"decided by the servicebus"  Do you think it'd be less ambiguous to have (both here and elsewhere) "default values as defined by the service"?

(Feel free to chime in here as well @yunhaoling and @annatisch )

Reply:
`defined` sounds better to me

Reply:
changed to "as defined by the service"